packages/babel-plugin-i18n-calypso/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@automattic/babel-plugin-i18n-calypso",
+  "version": "1.0.0",
+  "description": "A Babel plugin to generate a POT file for translate calls",
+  "main": "src/index.js",
+  "dependencies": {
+    "@babel/runtime": "^7.0.0",
+    "gettext-parser": "^1.3.1",
+    "lodash": "^4.17.10"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/traverse": "^7.0.0"
+  },
+  "peerDependencies": {
+    "@babel/core": "^7.0.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [],
+  "author": "Alex Kirk <alex.kirk@automattic.com> (https://automattic.com)",
+  "license": "GPL-2.0+"
+}
```

Comment:
Ran `npm publish --dry-run` from `packages/babel-plugin-i18n-calypso/`, seems to work :+1: